src/cqparts_motors/stepper.py

```diff
@@ -0,0 +1,185 @@
+
+
+import cadquery as cq
+
+import cqparts
+from cqparts.params import *
+from cqparts.display import render_props, display
+from cqparts.constraint import Fixed, Coincident
+from cqparts.constraint import Mate
+from cqparts.utils.geometry import CoordSystem
+
+
+class Axle(cqparts.Part):
```

Comment:
suggested change to `_Axle`

Reply:
I think this should be moved out as a Shaft, there are lots of different shaft set ups that we can use. And having mattes provided by the shaft would be helpful with larger constructs.